e2e/framework/wrappers.go

```diff
@@ -26,13 +26,14 @@ import (
 	"testing"
 
 	"github.com/go-redis/redis"
+	redisV8 "github.com/go-redis/redis/v8"
 
 	"github.com/topfreegames/maestro/e2e/framework/maestro"
 
 	"k8s.io/client-go/kubernetes"
 )
 
-func WithClients(t *testing.T, testCase func(apiClient *APIClient, kubeClient kubernetes.Interface, redisClient *redis.Client, maestro *maestro.MaestroInstance)) {
+func WithClients(t *testing.T, testCase func(apiClient *APIClient, kubeClient kubernetes.Interface, redisClient *redis.Client, redisClientV8 *redisV8.Client, maestro *maestro.MaestroInstance)) {
```

Comment:
Can we set a `TODO` here to remove the "old" client when all the ports are migrated into the v8?